### DISCUSSION TITLE
How to change the alignment for the MenuBar mobile collapsed menu?

### DISCUSSION THREAD

**Discussion by richcolemantx** [primefaces/community#893] [PrimeVue] [answered]
By default, this is on the left and I would like it on the right so I can put a logo/company on the left.

![image](https://github.com/primefaces/community/assets/91837210/1e7b9759-2d3a-47a6-89c7-d95c2e1cd49b)

Also, it appears to only collapse at below small.  How can I change the breakpoint that the menu collapses?

**Comment by tugcekucukoglu** [comment, marked as answer]
It can be changed like this: 


```
<style>
.p-menubar-mobile {
  justify-content: end;
}
</style>
```

**Reply by richcolemantx** [reply]
Well, yes, that actually worked for that example.  Apparently, since I have #end items, that solution doesn't apply.  How can I do the same for this example?  Sorry for the confusion.

https://tailwind.primevue.org/menubar/#template

**Comment by tugcekucukoglu** [comment]
Adding [Tailwind order classes](https://tailwindcss.com/docs/order) to `button` option in preset could solve this issue.
